### ISSUE TITLE
DwC field scientificNameID is not used at all

### ISSUE THREAD

**Issue by bart-v** [gbif/portal-feedback#1452]
Example
https://www.gbif-uat.org/occurrence/search?dataset_key=740cf4e0-37ca-4389-ba8f-4e1bc5177893&taxon_key=5401803

Lists the records as Oligochaeta and appends the authority "K.Koch" just like that.
That makes these marine occurrences terrestrial plants... 

While a scientificNameID urn:lsid:marinespecies.org:taxname:2036 is provided, that can be resolved to the animal class Oligochaeta.

This is a missed chance to fix homonyms in an easy way...

**Comment by MortenHofft** [comment]
Above link is using UAT (User acceptance testing). The result might be the same, but test environment and production environment is likely to be different much of the time.

**Comment by MattBlissett** [comment]
(UAT and the production environment are usually very similar for record interpretation.)

@mdoering can help explain what's happening here. I can see we have the name from the WoRMS checklist: https://www.gbif.org/species/105760798 but it isn't linked in to our taxonomic backbone.  [This one](https://www.gbif.org/species/101442716) from a different checklist is, but it loses the author.

We don't yet match using the scientificNameId, we should look to support this for the ingestion pipeline rewrite (in progress this year).

Adding a kingdom will make this match to the correct name.

**Comment by mdoering** [comment]
yes, scientificNameId is pretty much ignored in both occurrence and checklist processing.
With a a scientificNameId value from occurrences pointing to another checklist or even something outside of GBIF indexed data it will not become a simple exercise. All occurrences must link to a Backbone species, not some other checklist. For that to happen the backbone would a) have to have such a species (with that authorship) and b) know about the global scientificNameIds used in other lists. Maybe sth for when we have switched the backbone to use CoL+

**Comment by ManonGros** [comment]
Most of the links on this issue are deprecated.
Plus, as far as I understand, this is not something that we can fix. Could we close this issue?

**Comment by bart-v** [comment]
This is not about dead links, but about the fileld dwc:scientificNameID being ignored by GBIF
I thinks it's a very important issue

**Comment by albenson-usgs** [comment]
Agree this is an important issue, especially for OBIS node contributions. This really is a missed opportunity for GBIF as OBIS nodes take great care in assigning an appropriate scientificNameID to each occurrence. Would hate to see any records from the OBIS-USA node end up as terrestrial species when we've taken the time to provide the marine representation.

**Comment by ManonGros** [comment]
In that case, should it be an issue for the CoL+? https://github.com/Sp2000/colplus

**Comment by mdoering** [comment]
I am wondering about a few things here:

1. why does the name have a classification? [dwc:scientificNameID](https://dwc.tdwg.org/terms/#scientificNameID) should point to nomenclatural information. Taxon concepts and classifications would be [dwc:taxonConceptID](https://dwc.tdwg.org/terms/#taxonConceptID) or even just dwc:taxonID

2. We never dealt with DwC archives pointing to external information. All archive IDs can be resolved locally within the archive. This is not true for dwc:scientificNameID

3. For resolving external IDs there is no standard format, protocol or anything alike. Its quite a burden to know all variations in advance and issue http calls to resolve each ID.

4. Is there really extra information in the linked name data that would help us to better interpret the name & its classification? Isnt all that information already given in DwC? 


Looking at one of the Oligochaeta Koch examples I see the taxonomic dwc occurrence information is very sparse: https://www.gbif-uat.org/occurrence/1324564024
It is just the name, not even a rank, kingdom or anything else. The ID would have made a difference here. But would it be difficult to enrich the occurrence data?

http://lsid.info/urn:lsid:marinespecies.org:taxname:2036
```
<?xml version="1.0"?><rdf:RDF
	xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
	xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:dcterms="http://purl.org/dc/terms/"
	xmlns:dwc="http://rs.tdwg.org/dwc/terms/"

>
	<rdf:Description rdf:about="urn:lsid:marinespecies.org:taxname:2036">
		<dc:type>ScientificName</dc:type>
		<dc:date>2019-10-03</dc:date>
		<dc:subject><![CDATA[Oligochaeta Grube, 1850]]></dc:subject>
      <dc:title><![CDATA[Oligochaeta]]></dc:title>
      <dc:relation><![CDATA[http://www.marinespecies.org/aphia.php?p=taxdetails&amp;id=2036]]></dc:relation><dc:creator><![CDATA[Timm, Tarmo]]></dc:creator><dc:creator><![CDATA[van Haaren, Ton]]></dc:creator><dc:identifier>urn:lsid:marinespecies.org:taxname:2036</dc:identifier>
      <dc:publisher>World Register of Marine Species (WoRMS)</dc:publisher>
      <dc:license>http://creativecommons.org/licenses/by/4.0/</dc:license>
	  <dc:language>en</dc:language>
<dcterms:bibliographicCitation><![CDATA[WoRMS (2019). Oligochaeta. Accessed at: http://www.marinespecies.org/aphia.php?p=taxdetails&id=2036 on 2019-10-03]]></dcterms:bibliographicCitation><dcterms:created>2004-12-21T16:54:05+01:00</dcterms:created>
      <dcterms:modified>2017-06-01T14:33:21+01:00</dcterms:modified>
<dcterms:rightsHolder>WoRMS Editorial Board</dcterms:rightsHolder>
<dwc:kingdom>Animalia</dwc:kingdom>
      <dwc:phylum>Annelida</dwc:phylum>
      <dwc:class>Clitellata</dwc:class>
      <dwc:order></dwc:order>
      <dwc:family></dwc:family>
      <dwc:genus></dwc:genus>
      <dwc:subgenus></dwc:subgenus>
      <dwc:specificEpithet></dwc:specificEpithet>
      <dwc:infraspecificEpithet></dwc:infraspecificEpithet>
      <dwc:taxonRank>subclass</dwc:taxonRank>
      <dwc:ScientificName><![CDATA[Oligochaeta Grube, 1850]]></dwc:ScientificName>
	  <dwc:scientificNameAuthorship><![CDATA[Grube, 1850]]></dwc:scientificNameAuthorship>
	  <dwc:taxonomicStatus><![CDATA[accepted]]></dwc:taxonomicStatus>
<dwc:namePublishedIn><![CDATA[Grube, Adolf Eduard. (1850). Die Familien der Anneliden. <em>Archiv für Naturgeschichte, Berlin.</em> 16(1): 249-364.]]></dwc:namePublishedIn>
  <dwc:namePublishedInYear>1850</dwc:namePublishedInYear><dwc:scientificNameID rdf:resource="urn:lsid:marinespecies.org:taxname:2036" />
     <dwc:parentNameUsageID rdf:resource="urn:lsid:marinespecies.org:taxname:14165" />	</rdf:Description>
</rdf:RDF>
```

**Comment by mdoering** [comment]
The point of (dwc) archives is that it is NOT linked data. But if we had a (WoRMS) checklist that defined those IDs we could cross reference them so the taxonomic information would not have to be repeated in the occurrences.

**Comment by mdoering** [comment]
> In that case, should it be an issue for the CoL+? https://github.com/Sp2000/colplus

To some degree yes, but it is primarily an Occurrence interpretation issue

**Comment by bart-v** [comment]
To answer your questions @mdoering 

1. Point taken. WoRMS does not make a good distinction between names and concepts. This is work in progress.
2. We can't make all providers of Occurrence data responsible for the names, and ask them to generate a Darwin Core Taxon extension
3. True, so we need some custom code, no big deal right?
4. Yes, there is. OBIS even advises to leave out any Taxon related field and focus on the scientificNameID 

You have a WoRMS checklist that defines those: https://www.gbif.org/dataset/2d59e5db-57ad-41ff-97d6-11f5fb264527

**Comment by mdoering** [comment]
I think referring to a known checklist like WoRMS and reusing their taxonIDs makes a lot of sense and GBIF should support that in the long run. @timrobertson100 maybe the pipelines project can be a good way to include such a taxonID lookup. 

Still there are many detail questions, I have a few popping up immediately:
 - should an occurrence dataset specify which taxonomic authority checklist in GBIF they are based on? I reckon there can even be multiple? 
 - what about different versions used between checklist and occurrence dataset?
 - if we rely on globally unique ids instead of selected checklists, how do we know which checklist is the authority in case several checklists use these ids?

**Comment by timrobertson100** [comment]
Thanks @bart-v @albenson-usgs 

Currently `dwc:scientificNameID` just passes through ignored - but that just reflects the state of play when that codebase was written and the term was not well used. That is not the case today, and I agree GBIF should make use of it for cases when it clearly identifies e.g. WORMS, Index Fungorum records - especially as it is the OBIS recommendation to publishers.

I will move this issue into the gbif pipelines project, where we'll implement it working through the issues @mdoering rasies. All effort right now is on making the new ingestion pipeline live.
